Python/Product/TestAdapter/testlauncher.py

```diff
@@ -177,7 +177,8 @@ def collectwapper():
             notfound = getattr(collector, '_notfound', [])
             if notfound:
                   for arg, exc in notfound: 
-                      line = "(no name {!r} in any of {!r})".format(arg, exc.args[0])
+                      arg_zero = exc[0] if isinstance(exc, list) else exc.args[0]
+                      line = "(no name {!r} in any of {!r})".format(arg, arg_zero)
```

Comment:
"exc[0] if isinstance(exc, list) else exc.args[0]"  wow this looks like python

Reply:
It's a python file? Were you commenting on how different python looks than C#?